frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -511,6 +512,7 @@ export const experimentLogic = kea<experimentLogicType>([
                 openPrimaryMetricModal: (_, { index }) => index,
                 closePrimaryMetricModal: () => null,
                 updateExperimentGoal: () => null,
+                setEditingPrimaryMetricIndex: (_, { index }) => index,
```

Comment:
this is a bit confusing to me, how does the exposure metric relate to the primary metric? And now do we have separate exposure for each primary metric?